sentry-android-core/src/main/java/io/sentry/android/core/IActivityFramesTracker.java

```diff
@@ -0,0 +1,22 @@
+package io.sentry.android.core;
+
+import android.app.Activity;
+import io.sentry.protocol.MeasurementValue;
+import io.sentry.protocol.SentryId;
+import java.util.Map;
+import org.jetbrains.annotations.NotNull;
+import org.jetbrains.annotations.Nullable;
+
+public interface IActivityFramesTracker {
```

Comment:
I believe we don't need an interface, but rather just a boolean check.
See `ActivityFramesTracker#isFrameMetricsAggregatorAvailable`, you can leverage this method.
Adding an interface and not having different implementations just add overhead to the SDK.

Reply:
The `NoOp` impl is quite a common thing in the SDK. I like the approach.

Reply:
Although I would have preferred the interface solution (imagine having a tracker impl for Android, as well as a pure Java one) I removed it for now because as @romtsn pointed out in another comment init is a bit tricky, as right now `ActivityFramesTracker` is both used by an integration and an events processor, making it difficult to swap it out "mid-flight".